mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1126,19 +1126,41 @@ def _generate_model_predictions(self):
         ):
             y_pred_list = []
             pred_latencies = []
+            num_tokens_list = []
             X_copy = self.X.copy_to_avoid_mutation()
 
+            import tiktoken
+
+            encoding = tiktoken.get_encoding("cl100k_base")
+
+            def compute_num_tokens(y_pred):
```

Comment:
Should we scope the tiktoken import and the encoding declaration to within `compute_num_tokens` since they are unused elsewhere?